### PR TITLE
[core] Fix checkstyle for Client

### PR DESCRIPTION
**Pull request by risdenk**
Relates to #895

### CODE REVIEW

**Comment by risdenk** [comment]
Tested with: `mvn -pl core -am validate | grep "ycsb/Client"`

**Comment by risdenk** [comment]
Required by #920

File: core/src/main/java/com/yahoo/ycsb/Client.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (c) 2010 - 2016 Yahoo! Inc. All rights reserved.
```
Comment (busbey) [inline core/src/main/java/com/yahoo/ycsb/Client.java]: looks like copyright date needs to change.

Comment (risdenk) [inline core/src/main/java/com/yahoo/ycsb/Client.java]: Ah yup I'll fix that.

**Pull request merged**